pkg/kubelet/kubelet.go

```diff
@@ -2025,11 +2025,11 @@ func (kl *Kubelet) handleMirrorPod(mirrorPod *v1.Pod, start time.Time) {
 func (kl *Kubelet) HandlePodAdditions(pods []*v1.Pod) {
 	start := kl.clock.Now()
 	sort.Sort(sliceutils.PodsByCreationTime(pods))
+	// Responsible for checking limits in resolv.conf
```

Comment:
Nit: Can you make it clear that these limits do not have anything to do with individual pods/the existence of an individual pod has no impact on these limits?